packages/styles/src/variables/components/_toasts.scss

```diff
@@ -1,18 +1,19 @@
-@use './../color';
+@use 'sass:map';
 
-// Toasts
+@use './../spacing';
+@use './../type';
+@use './../components/alert';
 
-$toast-max-width: 350px !default;
-$toast-padding-x: 0.75rem !default;
-$toast-padding-y: 0.25rem !default;
-$toast-font-size: 0.875rem !default;
-$toast-color: null !default;
-$toast-background-color: rgba(color.$white, 0.85) !default;
-$toast-border-width: 1px !default;
-$toast-border-color: rgba(color.$black, 0.1) !default;
-$toast-border-radius: 0.25rem !default;
-$toast-box-shadow: 0 0.25rem 0.75rem rgba(color.$black, 0.1) !default;
+$toast-width: 25rem;
+$toast-width-sm: 21rem;
 
-$toast-header-color: color.$gray-60 !default;
-$toast-header-background-color: rgba(color.$white, 0.85) !default;
-$toast-header-border-color: rgba(color.$black, 0.05) !default;
+$toast-margin: spacing.$spacer;
+$toast-padding-x: spacing.$size-regular;
+$toast-padding-y: spacing.$size-small-regular;
+$toast-font-size: type.$font-size-tiny;
+$toast-heading-font-size: type.$font-size-small;
+$toast-hr-margin-top: spacing.$size-micro;
+$toast-hr-margin-bottom: spacing.$size-small-regular;
+$toast-icon-size: spacing.$size-big;
+
+$toast-variants: alert.$alert-variants;
```

Comment:
This means, we should move the deprication message for the toast/notification variants up on the demo app.
Because we can not deprecate colors for the toast but not for the alert.

![image](https://user-images.githubusercontent.com/9716662/235074797-003ee5d5-f2f6-4f2a-bbb3-8930f45c0ca2.png)

The deprecation warning should be included before the "Toast / Small notification bar" heading.